README.md

```diff
@@ -34,7 +34,8 @@
 - Built-in graphics library
 - Easy cross compilation
 - REPL
-- Built-in ORM
+- [Built-in ORM](https://github.com/tarunama/v/blob/master/examples/database/pg/customer.v)
```

Comment:
I would not link the pg example because the docs say that only sqlite is supported in ORM rn.
>V has a built-in ORM (object-relational mapping) which supports SQLite, and will soon support MySQL, Postgres, MS SQL, and Oracle.
https://github.com/vlang/v/blob/master/doc/docs.md#orm